### PR TITLE
Push changes 20240527

### PR DESCRIPTION
**Pull request by Grand-Thibault**

### CODE REVIEW

File: docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md [hunk 1]
```diff
@@ -35,35 +49,35 @@ ToDo: Add the whole list and link to the SharePoint page -> Problem... not every
...
 
 #### Alignment with Other Committees
 
-The Committee Group coordinates its activities with other committees to ensure a unified strategy and direction.
+The Committee coordinates its activities with other committees to ensure a unified strategy and direction.
 
 #### Assigning and Mentoring Expert Groups
 
-The group is responsible for assigning tasks to expert groups and providing support and mentoring to ensure their success.
+The Committee is responsible for assigning tasks to expert groups and providing support and mentoring to ensure their success.
```
Comment (jSchuetz88) [inline docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md:72]: Two points:
 - Is the Committee actually responsible, or able to assign tasks?
 - maybe add something like "*[...] assigning tasks to expert groups within their area / domain of responsibility [...]*"

File: docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md [hunk 2]
```diff
@@ -96,30 +108,27 @@ The committee is responsible for developing and maintaining the organizations in
 
 The aim of the expert groups within the Catena-X Automotive Network e.V. is to develop results together with the members in line with the association's strategic direction. The expert groups are managed by the related committee. The expert groups don't exist forever. They work on dedicated topics, present the results to the committee and are then retired.
 
+![Expert Group Lifeline](../assets/expert-group-life-line.drawio.svg)
+
+The focus of the expert group must be on the strategic goal roadmap of Catena-X. The association focuses on implementation and realisation in the expert groups and, via the committee, checks the expert groups for the delivery of results and not for exchange.
```
Comment (jSchuetz88) [inline docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md:113]: Most of the times, "committee" was written in upper case. I'm fine with either way, but should by consistent.

File: docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md [hunk 3]
```diff
@@ -131,45 +140,92 @@ There is already a list of groups that have been established and are currently a
...
 :::info
 ToDo: Add the whole list and link to the SharePoint page -> Problem... not everybody has access to this page
+List should have one single point of truth to avoid double maintenance
 :::
 
-### Tasks
+### Expert Group Election
 
-### Powers
+Association members can join any expert groups by applying for open seats. The recommended size for expert groups is 5 to 15 core team members. Less core team members might indicated that the topic is not relevant enough to the assosciation and cause the expert group not being established. More core team members would make the expert group to inflexible and a split of the expert group might be considered.
```
Comment (jSchuetz88) [inline docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md:148]: typo: assosciation

File: docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md [hunk 4]
```diff
@@ -131,45 +140,92 @@ There is already a list of groups that have been established and are currently a
...
-Individuals or organizations interested in forming an Expert Group can reach out to the Catena-X Association or any member of the committee. The request is then added to the committee's weekly agenda for consideration.
+![ExpertGroup establishment](../assets/expert-group-establishment-process.drawio.svg)
 
-**Introduction by Expert Group Mentor**
-An assigned Expert Group Mentor serves as the Single Point of Contact (SPOC) for requestors and guides them through the initiation process. The Mentor explains the process and provides necessary documents to the requestors.
+#### Request Submission
 
-**Expert Group Profile Presentation**
-The Mentor walks through each section of the Expert Group Profile in detail, without the use of slides. This ensures a comprehensive understanding of the group's objectives and scope.
+Individuals or organizations interested in forming an Expert Group can reach out to the Catena-X Association. The request is then going to be assigned to an committe. The request is then added to the committee's agenda for validation.
```
Comment (jSchuetz88) [inline docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md:167]: "Expert group" is mostly written in lower case. Personally I prefer the Upper Case-variant. But it should be consistent.

File: docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md [hunk 5]
```diff
@@ -131,45 +140,92 @@ There is already a list of groups that have been established and are currently a
...
 
-**Bi-weekly Slot Allocation**
-Expert Group slots are scheduled bi-weekly on Wednesdays from 1:00 to 1:30 PM. Each slot includes a 10-minute presentation, followed by a 15-minute Q&A/feedback session and a 5-minute buffer.
+#### Initial validation
 
-**Application Phase**
-The application phase typically lasts around two weeks. During this time, the Mentor may need to filter the list of applicants according to predefined criteria, such as ensuring an even distribution between company areas and allowing only one member per company. If there are sufficient applicants, an optional review group may be created.
+The committee is going to do a initial validation of the request. If the request is deemed valid, one of the committee members is going to be assigned to the request to serve as the Single Point of Contact (SPOC) for requestors and guides them through the initiation process. The mentor explains the process and provides necessary documents to the requestors.
```
Comment (jSchuetz88) [inline docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md:171]: General suggestion: maybe we should make acronyms used for the first time italic?

``` *Single Point of Contact* (SPOC) ```

-> *Single Point of Contact* (SPOC)

File: docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md [hunk 6]
```diff
@@ -131,45 +140,92 @@ There is already a list of groups that have been established and are currently a
...
+
+#### Application Phase
+
+Validated request are then taken over by the association which is then starting the application phase for the expert group. The application phase typically lasts around two to four weeks.
+
+#### Expert group composition
+
+All applicants are going to be reviewed by the mentor. Based on predefined criteria the members for the expert group are going to be selected. If there are sufficient applicants, an optional review group may be created (see [Expert group election](#expert-group-election)).
+
+#### Expert group approval
```
Comment (jSchuetz88) [inline docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md:189]: Suggestion: Maybe we need to add a sub-section or information, that expert groups are temporarily and must provide a clear timeline for their task to be achieved.

File: docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md [hunk 7]
```diff
@@ -131,45 +140,92 @@ There is already a list of groups that have been established and are currently a
...
 
-### Decision Making & Communication
+### Expert Group maintenance over the life time
 
-### Escalation Paths
+The composition of the expert groups is not static and is expected to change over the life time of the expert group. In any case people that want to join an existing expert group need to be approved the same way, as applicants for new expert groups. This means they need to be approved by the mentor of the expert group and the managing director value and technology creation.
+
+#### Replacement of expert group members
+
+If a expert group member is stepping down from its position, it can not be handed over to another employee of the same company. The replacement needs to be approved.
```
Comment (jSchuetz88) [inline docs/working-model/02-organizational-structure/overview-roles/catena-x-roles.md:213]: approved by whom and how? The Committee? The Expert Group? and as a vote? or decision by the leads?

File: docs/working-model/03-process-from-idea-to-production/03-01-process-from-idea-to-production.md [hunk 8]
```diff
@@ -11,6 +11,118 @@ We use the Committees and Expert Groups of the Catena-X e.V. to align, develop a
...
+### Check strategic fit
+
+* Responsible: Topic owner
+* Description: Once the feature proposal contains at least the minimal feature content, its strategic fit is going to be evaluated by the topic owner
+* Possible outcomes:
+  * Feature proposal fit strategy: If the feature proposal strategically fits it is going to be added to the Inbox.
+  * Strategic fit cant be assest: If the strategic fit cant be assest by the topic owner the feature proposal is being send back to the feature author with the request to further refine the feature proposal.
+  * Feature proposal does NOT fit: If the topic owner assess the feature proposal as not fitting the strategy, the feature proposal is going to be rejected. In this case the feature proposal is not going to be implemented.
+
+### Refine feature to meet DoE
```
Comment (jSchuetz88) [inline docs/working-model/03-process-from-idea-to-production/03-01-process-from-idea-to-production.md:45]: Please introduce the acronym, as it is used for the first time here.

File: docs/working-model/03-process-from-idea-to-production/03-01-process-from-idea-to-production.md [hunk 9]
```diff
@@ -194,8 +295,8 @@ Defects or unexpected behavior must be reported as bugs in the [sig-release repo
 
 There are various testing artifacts, that are either managed in Tractus-X GitHub or the Test Management Tool.
 
-- In **GitHub** we manage the different user journys and related business scenarios as .md files.
-- In the **Test Management Tool** we manage test cases, test sets (opt.), test exectuions as well as test plans and reports.
+* In **GitHub** we manage the different user journys and related business scenarios as .md files.
```
Comment (jSchuetz88) [inline docs/working-model/03-process-from-idea-to-production/03-01-process-from-idea-to-production.md:298]: typo: -> journys

**Review by jSchuetz88 — CHANGES_REQUESTED** [review]
Great work! 
Some of my remarks may require their own issue.
And most of the remarks were just minor typos and/or inconsistencies.

Only two remarks I consider as an issue: 
- the images with a transparent background and black text won't be readable within the dark template (e.g. From Idea to Release Item.drawio.svg ). Please use a white background, so that the images will work in both themes. Images in general should be optimized for the light theme, but also be readable with the dark theme.
- "From Idea to Release Item.drawio.svg" Filenames shouldn't contain whitespaces or upper cases. Plase rename in e.g. "from-idea-to-release-item.drawio.svg" (see kebab-case convention)

**Review by stephanbcbauer — APPROVED** [review]
LGTM -> All change requests will be worked in into the private repository by @Grand-Thibault

**Pull request merged**